### DISCUSSION TITLE
Basics for a *Good* Image Protocol

### DISCUSSION THREAD

**Discussion by arthsmn** [kovidgoyal/kitty#6936] [General]
I saw [this issue](https://gitlab.freedesktop.org/terminal-wg/specifications/-/issues/26) talking about the specifics of a good image protocol. They say there's no such protocol right now, so I would like to know in which parts do the kitty's graphics protocol falls short and (hopefully) understand how can it be improved.

**Comment by kovidgoyal** [comment]
On Thu, Dec 21, 2023 at 01:03:16AM -0800, Arthur wrote:
> I saw [this issue](https://gitlab.freedesktop.org/terminal-wg/specifications/-/issues/26) talking about the specifics of a good image protocol. They say there's no such protocol right now, so I would like to know in which parts do the kitty's graphics protocol falls short and (hopefully) understand how can it be improved.

Ask them. In my opinion, they are idiots.

**Comment by sergei-grechanik** [comment]
With the Unicode placeholder extension, kitty's graphics protocol is actually pretty close to what's described there.
